src/qml/imports/Theme/Theme.qml

```diff
@@ -63,5 +63,10 @@ QtObject {
       var path = '/themes/' + theme + '/nodpi/' + name + '.svg';
       return path;
     }
+
```

Comment:
```suggestion

function toInlineStyles(styleProperties) {
  var styles = ''
  
  for (var property in styleProperties) {
    var value = styleProperties[property];
    style += property
    style += ': '
    style += typeof value == 'color'
      ? colorToHtml(value)
      : value
    style += ';'
  }
  
  return styles;
}

```
Have not tested, but I would suggest setting styles like:
```
  'String with <span style="%1">styles</a>'.arg(toInlineStyles({
    color: Theme.mainColor,
    weight: checkboxComponent.checked ? 'bold' : 'normal'
  }))
```

Reply:
Sounds good.
Can you push this approach directly to this branch?